CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 
 [Unreleased changes](./UNRELEASED.md).
 
+# 33.0.0
+
+**Breaking:**
+  - The minimum deployment target is now iOS 12.0 (previously iOS 11.0).
+
```

Comment:
This should be in unreleased

Reply:
Ah yeah that's a side effect of me being halfway through the release before starting this.

Reply:
Fixed